lib/hocon/impl/simple_config.rb

```diff
@@ -61,12 +61,12 @@ def self.find_key(me, key, expected, original_path)
     if v.value_type == ConfigValueType::NULL
       raise ConfigNullError.new(v.origin,
                                 (ConfigNullError.make_message(original_path.render,
-                                                              (not expected.nil?) ? ConfigValueType.value_type_name(expected) : nil)),
+                                                              (not expected.nil?) ? Hocon::ConfigValueType.value_type_name(expected) : nil)),
```

Comment:
In this file, on line 19, there is an alias that defines a local constant `ConfigValueType` to reference the fully-qualified `Hocon::ConfigValueType`.  That means that in this file we can refer to it as just `ConfigValueType`; so, this change is not harmful, but not necessary.  The important part is that we are calling the method as a class method rather than as an instance method, and here, we already were.